spring-xd-codec/src/main/java/org/springframework/xd/dirt/integration/bus/serializer/kryo/AbstractKryoCodec.java

```diff
@@ -38,17 +38,6 @@
  */
 public abstract class AbstractKryoCodec<T> extends AbstractCodec<T> {
 
-	protected final static int FILE_REGISTRATION_ID = 40;
```

Comment:
The intention here is that these would be done by the user should they want to have IDs instead of strings for classnames for these types?